tests/test_other.py

```diff
@@ -9009,6 +9009,10 @@ def ok(args, filename='hello_world.cpp', expected='hello, world!'):
     ok(required_flags, filename='hello_world_main_loop.cpp')
     # -O1 is ok as we don't run wasm-opt there (but no higher, see below)
     ok(required_flags + ['-O1'])
+    # Exception support shouldn't require changes after linking
+    ok(required_flags + ['-fexceptions'])
+    # longjmp support shouldn't require changes after linking
+    ok(required_flags + ['-sSUPPORT_LONGJMP=1'])
```

Comment:
longjmp is on by default, so I think this part is already tested, and not needed.

Reply:
Removed.